### PR TITLE
Updates to pull scrape items from MongoDB and run scraper on html and PDFs.

### PR DESCRIPTION
**Pull request by yahoo-che**
Ok made this PR into quite a few updates.
- Database methods `loadItemsToScrape` and `getUsersToNotify` actually pull from the the remote MongoDB.
- Pull MongoDB url from config file.
- Disabled notify call for now to focus on database and scraping.
- Moved `evaluate` method from scheduler into scraper
- Updated `isRelative` method for better detection of relative path urls since path may not start with nothing not just '.' or '/'
- Added `dedup` array util since state sites were returning some duplicate PDF urls.
- Trying a later version of `pdf.js` in `pdf-parse` (which hasn't been updated in 2 years)
- Catch any errors with `axios` and `pdf-parse`
- Added unit test for PDF scraper

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,21 +27,20 @@
...
     "diff": "^4.0.2",
+    "dotenv": "^8.2.0",
     "express": "^4.17.1",
     "html-to-text": "^5.1.1",
     "mocha": "^8.1.3",
-    "mongoose": "^5.10.5"
-  },
-  "devDependencies": {
-    "chalk": "^4.1.0",
-    "csv-parse": "^4.12.0",
```
Comment (adeolaawoyemi) [inline package.json:40]: @yahoo-che these are needed by the import script. That's why they're in the `devDependencies`.

File: package.json [hunk 2]
```diff
@@ -27,21 +27,20 @@
...
-    "pdf-parse": "^1.1.1",
     "mongoose": "^5.10.5",
     "nodemailer": "^6.4.11",
-    "pug": "^3.0.0"
```
Comment (adeolaawoyemi) [inline package.json:44]: I think this is needed too in the tests.

**Review by adeolaawoyemi — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (clayreimann) [reply package.json:44]: This is used in by express, also can be used to make the email templating easier/more flexible

**Review by adeolaawoyemi — APPROVED** [review]
👍

**Pull request merged**